docs/resources/service_principal.md

```diff
@@ -5,7 +5,7 @@ subcategory: "Security"
 
 Directly manage [Service Principals](https://docs.databricks.com/administration-guide/users-groups/service-principals.html) that could be added to [databricks_group](group.md) in Databricks workspace or account.
 
-To create service principals in the Databricks account, the provider must be configured with `host = "https://accounts.cloud.databricks.com"` on AWS deployments or `host = "https://accounts.azuredatabricks.net"` and authenticate using AAD tokens on Azure deployments
+To create service principals in the Databricks account, the provider must be configured with `host = "https://accounts.cloud.databricks.com"` on AWS deployments or `host = "https://accounts.azuredatabricks.net"` and authenticate using [AAD tokens](https://learn.microsoft.com/en-us/azure/databricks/dev-tools/api/latest/aad/) on Azure deployments
```

Comment:
https://registry.terraform.io/providers/databricks/databricks/latest/docs#special-configurations-for-azure rephrase to use this